### PR TITLE
Config

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

**Review by oriagranat9 — DISMISSED** [review]
looks good to me would also like @Bankst review before merging

File: chameleon-server/src/main/java/com/chameleonvision/common/configuration/CameraConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+public class CameraConfiguration {
+    private static final Logger logger = new Logger(CameraConfiguration.class, LogGroup.Camera);
+
+    public String name = "";
+    public String nickname = "";
+    public double FOV = 70;
+    public StreamDivisor divisor = StreamDivisor.NONE;
```
Comment (Bankst) [inline chameleon-server/src/main/java/com/chameleonvision/common/configuration/CameraConfiguration.java]: `StreamDivisor` needs to be removed. It's part of a pipeline.

File: chameleon-server/src/main/java/com/chameleonvision/common/configuration/NetworkConfig.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+import com.chameleonvision.common.networking.NetworkMode;
+
+public class NetworkConfig {
+    public int teamNumber = 1577;
+    public NetworkMode connectionType = NetworkMode.DHCP;
+    public String ip = "";
+    public String gateway = "";
+    public String netmask = "";
+    public String hostname = "Chameleon-vision";
```
Comment (Bankst) [inline chameleon-server/src/main/java/com/chameleonvision/common/configuration/NetworkConfig.java]: lower-case the hostname

File: chameleon-server/src/main/java/com/chameleonvision/common/configuration/StreamDivisor.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.chameleonvision.common.configuration;
```
Comment (Bankst) [inline chameleon-server/src/main/java/com/chameleonvision/common/configuration/StreamDivisor.java]: This class should be in the `common.vision` package, as it relates to the pipeline.

**Review by Bankst — CHANGES_REQUESTED** [review]

**Pull request merged**